src/main/kotlin/no/nav/syfo/Environment.kt

```diff
@@ -5,9 +5,11 @@ import java.lang.RuntimeException
 data class Environment(
     val applicationPort: Int = getEnvVar("APPLICATION_PORT", "8080").toInt(),
     val jwkKeysUrl: String = getEnvVar("JWKKEYS_URL", "https://login.microsoftonline.com/common/discovery/keys"),
-    val clientId: String = getEnvVar("CLIENT_ID"),
-    val jwtIssuer: String = getEnvVar("JWT_ISSUER"),
-    val appIds: List<String> = getEnvVar("ALLOWED_APP_IDS")
+    val clientId: String = getEnvVar("CLIENT_ID", ""),
```

Comment:
Ønsker vi egentlig å ha defaultverdier for disse..? Hvis man glemmer å sette disse i naiserator-filen vil det kanskje ta lenger tid å finne ut hvorfor ting feiler når man har en defaultverdi enn hvis de bare mangler..?

Reply:
Enig, ikkje har default her, heller at vi krasjer

Reply:
nei skal ikke ha default. det er en miss, bra du så den!